### PR TITLE
feat: upgrade to consola v3.x prerelease

### PR DESCRIPTION
**Pull request by danielroe**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This updates us to new consola v3 for testing on the edge prior to including in Nuxt v3.4 feature release. 🔥

👉 Check out the prerelease v3 release changelogs at https://github.com/unjs/consola/releases.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/nuxi/src/cli.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ consola.wrapAll()
 
 // Filter out unwanted logs
 // TODO: Use better API from consola for intercepting logs
-const wrapReporter = (reporter: ConsolaReporter) => <ConsolaReporter> {
+// TODO: export type from Consola
+const wrapReporter = (reporter: Consola['_reporters'][0]) => <Consola['_reporters'][0]> {
```
Comment (danielroe) [inline packages/nuxi/src/cli.ts:49]: cc: @pi0

**Comment by danielroe** [comment]
Merging to test on edge with named export.

@pi0 - what do you think about keeping as default export until more of ecosystem has upgraded?

**Comment by paulvonber** [comment]
Same here:
image


Nuxt: v3.4.0
Nodejs: v16.17.0
pnpm: v8.2.0

**Comment by danielroe** [comment]
See https://github.com/nuxt/nuxt/issues/20209.

**Pull request merged**